Dockerfile

```diff
@@ -1,39 +1,28 @@
-FROM java:7
+FROM java:8-jdk
 MAINTAINER Gabe Conradi <gabe@tumblr.com>
 
 RUN apt-get update && apt-get install -y zip unzip && rm -r /var/lib/apt/lists/*
 
-RUN useradd -Ur -d /opt/collins collins
-RUN for dir in /build /build/collins /var/log/collins /var/run/collins; do mkdir $dir; chown collins $dir; done
+ENV ACTIVATOR_VERSION 1.3.7
```

Comment:
Feels a little off to have this env var sitting around, since it's not used for anything except building the container.  What about moving it to the top of the `RUN` section, rather than having it as a global `ENV`?